opensrp-giz-malawi/src/test/java/org/smartregister/giz/fragment/ChildRegisterFragmentTest.java

```diff
@@ -0,0 +1,273 @@
+package org.smartregister.giz.fragment;
+
+import android.content.Intent;
+import android.view.View;
+import android.widget.EditText;
+
+import androidx.appcompat.widget.SwitchCompat;
+import androidx.fragment.app.FragmentActivity;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.rule.PowerMockRule;
+import org.powermock.reflect.Whitebox;
+import org.robolectric.util.ReflectionHelpers;
+import org.smartregister.Context;
+import org.smartregister.CoreLibrary;
+import org.smartregister.child.ChildLibrary;
+import org.smartregister.child.contract.ChildRegisterFragmentContract;
+import org.smartregister.child.cursor.AdvancedMatrixCursor;
+import org.smartregister.child.domain.ChildMetadata;
+import org.smartregister.child.presenter.BaseChildRegisterFragmentPresenter;
+import org.smartregister.child.provider.RegisterQueryProvider;
+import org.smartregister.child.util.Constants;
+import org.smartregister.child.util.Utils;
+import org.smartregister.commonregistry.CommonPersonObjectClient;
+import org.smartregister.commonregistry.CommonRepository;
+import org.smartregister.cursoradapter.RecyclerViewPaginatedAdapter;
+import org.smartregister.giz.BaseUnitTest;
+import org.smartregister.giz.R;
+import org.smartregister.giz.activity.ChildRegisterActivity;
+import org.smartregister.giz.presenter.ChildRegisterFragmentPresenter;
+import org.smartregister.immunization.ImmunizationLibrary;
+import org.smartregister.immunization.db.VaccineRepo;
+import org.smartregister.immunization.repository.VaccineRepository;
+import org.smartregister.immunization.util.IMConstants;
+import org.smartregister.location.helper.LocationHelper;
+import org.smartregister.service.AlertService;
+
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.when;
+
+@PrepareForTest({ImmunizationLibrary.class, Utils.class})
+public class ChildRegisterFragmentTest extends BaseUnitTest {
```

Comment:
Update test to follow the convention `test[MethodName][ExpectedResult][Parameters] (Parameters optional if not present)`

Reply:
Done